src/lhttpc_manager.erl

```diff
@@ -271,7 +271,7 @@ init(Options) ->
         true ->
             % Make sure that the ssl random number generator is seeded
             % This was new in R13 (ssl-3.10.1 in R13B vs. ssl-3.10.0 in R12B-5)
-            apply(ssl, seed, [crypto:rand_bytes(255)]);
+            apply(ssl, seed, [crypto:strong_rand_bytes(255)]);
```

Comment:
What about the possibility of `strong_rand_bytes` blowing up due to a lack of entropy?